github_rules/github_org_auth_modified.yml

```diff
@@ -0,0 +1,37 @@
+AnalysisType: rule
+Filename: github_org_auth_modified.py
+RuleID: GitHub.Org.AuthChange
+DisplayName: GitHub Org Authentication Method Changed
+Enabled: true
+LogTypes:
+  - GitHub.Audit
+Tags:
+  - GitHub
+Severity: Critical
+Description: Detects changes to GitHub org authentication changes.
+Runbook: Verify that the GitHub admin performed this activity and validate its use.
+Tests:
+  -
+    Name: GitHub - Authentication Method Changed
+    ExpectedResult: true
+    Log:
+      {
+        "actor": "cat",
+        "action": "org.saml_disabled",
```

Comment:
Can we add a `SummaryAttributes` field that includes at least `action`? Since hypothetically there could be multiple actions performed by the same actor rolled into one alert.

Reply:
Added